docker-compose.prod-img.yml

```diff
@@ -8,6 +12,7 @@ services:
     environment:
       ENVIRONMENT: production
       FLASK_APP: OpenOversight.app
+      OO_SERVICE_EMAIL: "freddy@lucyparsonslabs.com"
```

Comment:
This should not be here, but in the .env file

Reply:
On it.

Reply:
Which one tho? We have an `env` file in the migrations folder, but I do not see a `.env` file anywhere in the repository. I created one in the base directory, I'll verify that it works.